web/src/js/ducks/settings.js

```diff
@@ -0,0 +1,92 @@
+import { StoreCmds } from '../actions'
+import { addLogEntry } from './eventLog'
+
+export const WS_MSG_TYPE = 'settings'
+export const WS_MSG_CMD_RESET = 'reset'
+export const WS_MSG_CMD_UPDATE = 'update'
```

Comment:
I think it's better to have these constants somewhere, it's not a good idea to have magic strings in logic codes.